shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1587,11 +1572,9 @@ - (BOOL)isKeyboardNotificationForDifferentView:(NSNotification*)notification {
   if (isLocal && ![isLocal boolValue]) {
     return YES;
   }
+
   // Engine’s viewController is not current viewController.
```

Comment:
Remove the comment too; it doesn't make sense now that it's on the line that is both outcomes.

Reply:
Done.